biz-scenario/2 Table.sql

```diff
@@ -0,0 +1,31 @@
+use ArmyDB
+GO
+
+DROP table if EXISTS dbo.Army
+GO
+
+CREATE TABLE dbo.Army(
+    ArmyID int not null identity primary key,
+    FirstName varchar (20) not null,
+        CONSTRAINT c_Army_First_Name_can_not_be_blank CHECK (firstname <> ''),
+    LastName varchar (20) not null 
+        CONSTRAINT c_Army_Last_Name_can_not_be_blank CHECK (LastName <> ''),
+    SSN char (10) not null
+        constraint u_Army_SSN UNIQUE,
+        CONSTRAINT c_Army_SSN_must_have_a_dash CHECK  (ssn like'%-%'),
+    DateOfBirth date not null,
+    PlaceOfResidence varchar (35) not NULL
+        CONSTRAINT c_Army_place_of_residence_can_not_be_blank CHECK (PlaceOfResidence <> ''),
+    DateOfEnlistment date not null,
+       CONSTRAINT c_Army_age_must_be_older_than_seventeen CHECK (datediff (year, DateOfBirth, DateOfEnlistment) >= 17 ),
+    ServiceUnit VARCHAR (13) not NULL
```

Comment:
-- LL the varchar allowance should be 12 instead of 13 (Ground Force is 12 characters)